src/main/resources/anniv5Resources/localization/eng/magnetizepack/Keywordstrings.json

```
@@ -0,0 +1,11 @@
+[
+  {
+    "PROPER_NAME": "Magnetized",
+    "NAMES": [
+      "Magnetized",
+      "magnetized",
+      "magnetize"
+    ],
+    "DESCRIPTION": "When you play or discard a #yMagnetized card, #yMagnetize another random card (excluding #yUnplayable cards) in your hand. Some cards have special effects with #yMagnetized cards."
```

```suggestion
    "DESCRIPTION": "When you play or discard a #yMagnetized card, #yMagnetize another random card (excluding #yUnplayable cards) in your hand."
```
Bit too wordy, and it'll be self evident that it interacts with other things.